Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabOverrideLabelHandler.cpp

```diff
@@ -64,7 +67,9 @@ namespace AzToolsFramework::Prefab
 
         if (selectedItem == revertAction)
         {
-            // This is the place where revert override code will be added later.
+            AZStd::string_view relativePathFromEntity =
+                PrefabPropertyEditorNodes::PrefabOverrideLabel::RelativePath.ExtractFromDomNode(m_node).value_or("");
+            PrefabPropertyEditorNodes::PrefabOverrideLabel::RevertOverride.InvokeOnDomNode(m_node, relativePathFromEntity);
```

Comment:
It would be nice to have a way to send a message to the node or getting the node associated with the handler instead of caching it.

Reply:
I notice that the cached node is also needed for doing invoking 'RevertOverride'. It does check if the attribute exists in the node in `InvokeOnDomNode(m_node)`. Afaik there is no other place to bind the node to the handler, unless we cache the node in `SetValueFromDom()`.

fyi I followed the pattern that we used in [ContainerActionButtonHandler](https://github.com/o3de/o3de/blob/development/Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/ContainerActionButtonHandler.cpp#L36).

Reply:
Oh, I see, so we're already using this pattern elsewhere. Thanks for the info!